tests/helpers.py

```diff
@@ -1907,3 +1908,27 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def skipif_ocs_version(expr):
+    """
+    This is a decorator to skip the test if condition evaluates to
+    true based on expression
+
+    Args:
+        expr (str): condition for which we need to check,
+            eg: '<4.3', '>4.3'
+
+    Return:
+        decorated function
+    """
+    comparision_str = config.ENV_DATA['ocs_version'] + expr
+    # skip_this will be either True or False after eval
+    skip_this = eval(comparision_str)
+
```

Comment:
What about to use: @functools.wraps(func)

You can read a bit about it here:
https://lerner.co.il/2019/05/05/making-your-python-decorators-even-better-with-functool-wraps/

Reply:
@shylesh we use what Petr is talking about in our [retry module](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/retry.py) if you want another example.

Reply:
I think the better place for this is in conftest with pytest.fixture so we can avoid decorating it.

Reply:
I think that decorator is easier to use and easy to send parameter to it. How will you send the versions you would like to avoid to the fixture? Can you please elaborate more? Thanks